app.py

```diff
@@ -264,54 +259,87 @@ def custom_url_link(freq, name):
     return html.A(url.custom_text, href=url.custom_link)
 
 
-def varinfo(freq, name):
-    info = RemoteAPI(freq, name).info
-    try:                
-       start, end = (info[key] for key in ('start_date', 'end_date'))
-    except KeyError:
-       start, end = '', ''
-    return [f'Frequency: {freq}, variable name: {name}, ',
-            f'start: {start}, end: {end}, ', 
-             'custom URL: ', custom_url_link(freq, name)]
+def varinfo(freq, names):
```

Comment:
This is not correct. Varinfo interface is not to change - it returns a table of two columns for one variable. First column is keys like 'Variable', 'Latest', etc. + second column is values. Perhaps this is what you were asking before, but this implementation is not to specification.